frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -0,0 +1,294 @@
+      });
+
+      reloadApps();
+      setOpen(false);
+    } catch (error) {
+      handleRequestError(toast, "Failed to create app", error);
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+
+  return (
+    <>
+      <div className="h-full w-full max-w-screen-sm mx-auto flex flex-col justify-center">
+        <div className="flex flex-col items-center justify-center gap-6">
+          {!subscription &&
+            <>
+              <section className="text-center">
+                <h2 className="text-3xl font-semibold mb-2">
+                  ✨ Your Support Matters
+                </h2>
+                <p className="text-muted-foreground">
+                  Our open-source Lightning node is dedicated to enhancing the
+                  Bitcoin ecosystem by providing a reliable, efficient, and
+                  user-friendly platform for transactions. With your support, we can
+                  continue to innovate and expand our services.
+                </p>
+              </section>
+              <Card className="w-full">
+                <CardHeader>
+                  <CardTitle>Why Your Contribution Is Important</CardTitle>
+                </CardHeader>
+                <CardContent>
+                  <ul className="flex flex-col gap-2">
+                    <li className="flex flex-col ">
+                      <div className="flex flex-row items-center">
+                        <PlusCircle className="w-4 h-4 mr-2" />
+                        New Features
+                      </div>
+                      <div className="text-muted-foreground">
+                        Your support allows us to build and integrate new
+                        <ExternalLink className="underline" to="https://github.com/getAlby/hub/issues">
+                          features
+                        </ExternalLink>
+                      </div>
+                    </li>
+                    <li className="flex flex-col ">
+                      <div className="flex flex-row items-center">
+                        <RefreshCw className="w-4 h-4 mr-2" />
+                        Regular Updates
+                      </div>
+                      <div className="text-muted-foreground">
+                        Contributions fund ongoing maintenance and
+                        <ExternalLink className="underline" to="https://github.com/getAlby/hub/releases">
+                          improvements
+                        </ExternalLink>
+                      </div>
+                    </li>
+                    <li className="flex flex-col ">
+                      <div className="flex flex-row items-center">
+                        <Code className="w-4 h-4 mr-2" />
+                        Open-Source
+                      </div>
+                      <div className="text-muted-foreground">
+                        Keep Alby Hub open-source and free for all users
```

Comment:
I don't think we should put it this way - Alby Hub **MUST** be open source to be truly self-custodial, we cannot change that if users do not pay.

Reply:
I would just say Alby spends a lot of effort developing open source products and protocols. Your contribution helps us to continue to be able to do this.